### DISCUSSION TITLE
Use gulp prefix for all plugins

### DISCUSSION THREAD

**Discussion by gorankarlic** [gulpjs/gulp#2483] [Ideas]
# Situation

Currently it is very hard to keep track of gulp plugins in the `package.json`. IMHO:
- Modules widely used in a plugin-like manner (like `del`) should be also repackaged as a proper gulp plugin (e.g. `gulp-plugin-clean`)
- Modules mostly specific to gulp like `vinyl-` should be renamed / repackaged with the`gulp-stream-` (or similar) prefix 

Is there a good reason plugins are not "standardised" to use the `gulp-plugin-` prefix?

# Ideas?

Any ideas on how can we get an initiative started to standardise and clean up the naming within the gulp plugin ecosystem?

**Comment by phated** [comment]
This should be a discussion because that's a community thing. The gulp team doesn't provide any plugins.
